src/goto-programs/remove_exceptions.h

```diff
@@ -21,7 +21,15 @@ Date:   December 2016
 // Removes 'throw x' and CATCH-PUSH/CATCH-POP
 // and adds the required instrumentation (GOTOs and assignments)
 
-void remove_exceptions(symbol_tablet &, goto_functionst &);
-void remove_exceptions(goto_modelt &);
+enum class remove_exceptions_typest
+{
+  REMOVE_ADDED_INSTANCEOF,
+  DONT_REMOVE_INSTANCEOF,
+};
```

Comment:
Is this more than a Boolean?

Reply:
I favour keeping the enum since it makes the callsite clearer

Reply:
Ok, makes sense, but might warrant a comment to clarify that that's the sole intent?